pkg/ottl/contexts/internal/ctxprofile/profile.go

```diff
@@ -31,22 +31,8 @@ func PathGetSetter[K ProfileContext](path ottl.Path[K]) (ottl.GetSetter[K], erro
 		return accessSampleType[K](), nil
 	case "sample":
 		return accessSample[K](), nil
-	case "mapping_table":
-		return accessMappingTable[K](), nil
-	case "location_table":
-		return accessLocationTable[K](), nil
 	case "location_indices":
 		return accessLocationIndices[K](), nil
-	case "function_table":
-		return accessFunctionTable[K](), nil
-	case "attribute_table":
```

Comment:
Should we also remove the `attribute_indices`? Not sure if those indices would be useful without this removed table. Same question applies to other *indices that the lookup table is being removed: `location_indices`, `comment_string_indices`, `default_sample_type_string_index` (seems to use the `string_table`). 

If the plan is to abstract the internal format, I guess we wouldn't want them to be set manually, and some mechanism similar to the [attributes](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/39681) would be provided. Is that correct?